earth_enterprise/tutorial/FusionTutorial/download_tutorial.sh

```diff
@@ -3,6 +3,11 @@
 TMPFILE="/tmp/FusionTutorial-Full.tar.gz"
 SCRIPTDIR=`dirname "$0"`
 
+if [ "$EUID" != "0" ] ; then
+    echo `basename $0` "must be run as root"
```

Comment:
I think you should put everything in quotes here. We're expecting the script name not to have any special characters, but we can be sure it works by quoting.